src/OmniSharp.MSBuild/ProjectFile/PackageReference.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using NuGet.Packaging.Core;
+
+namespace OmniSharp.MSBuild.ProjectFile
+{
+    public class PackageReference : IEquatable<PackageReference>
```

Comment:
Note: NuGet's PackageReference is way different: https://github.com/NuGet/NuGet.Client/blob/4cccb13833ad29d6a0bcff055460d964f1b49cfe/src/NuGet.Core/NuGet.Packaging/PackageReference.cs. These aren't the same thing.